src/lib.rs

```diff
@@ -196,31 +197,24 @@ Location objects are composed of a single key which indicates the variant being
 pub enum Location {
     // clockwise: from -> to (degrees)
     /// Circular arc relative to the CUAS within which the UAS resides.
-    #[serde(rename = "arc")]
     Arc(Arc),
     /// Compass quadrant where the UAS has been observed.
-    #[serde(rename = "quad")]
-    Quad(Quad),
+    Quad { quad: Quad },
```

Comment:
I assume that, with the new tagging method, this would appear as **Quad:{quad:"North"}** in the format, am I right?

If this is the case then I don't see any problems with this new format. As for eliminating the renames, I believe this is a good decision.

Reply:
No, the location would rather appear as, for example,
```json
"$type": "Quad",
"quad": "West"
```
I tried to achieve this without breaking the Rust interface (via serde attributes) but I couldn't figure out how to do it.

Reply:
So it is essentially the same as the previous tagging (t-c) but with c being replaced by the members names and values of the type.

Reply:
Not exactly. This only applies to Quad and Bearing. Other variants are named structs, which are flattened onto the parent Location struct like so:
```json
"$type": "Location2d",
"lat": 0.2747,
"lon: -6.3664
```

Reply:
I understand